src/components/course/EnrollButton.jsx

```diff
@@ -33,20 +35,20 @@ export default function EnrollButton() {
     pacingType,
     courseUuid,
   } = activeCourseRun;
-  const [isEnrollDisabled, setIsEnrollDisabled] = useState(false);
-  const [emailOptIn] = useState(false);
-  const [enrollmentSubmitted, setEnrollmentSubmitted] = useState(false);
 
-  /*
-   * TODO: this behavior mimics the existing B2C enrollment flow. we will want
-   * to redirect learners to the basket flow instead of the track selection page
-   */
-  useEffect(() => {
-    if (enrollmentSubmitted) {
-      // redirect to track selection page
-      window.location.href = `${process.env.LMS_BASE_URL}/course_modes/choose/${key}`;
-    }
-  }, [enrollmentSubmitted]);
+  // This enrollment URL assumes that the learner has access to the course through their subscription license, and they
+  // are using the license to enroll.
+  // TODO: Update to conditionally use the DSC flow with a license uuid only when the learner is actually using a
+  // license.
+  const enrollOptions = {
+    license_uuid: subscriptionLicense.uuid,
+    course_id: key,
+    enterprise_customer_uuid: enterpriseConfig.uuid,
+    next: `${process.env.LMS_BASE_URL}/dashboard`,
```

Comment:
should the `next` URL be to the B2C dashboard? I believe it should be dropping the user into the courseware after a successful enrollment/DSC grant.

Reply:
I believe @iloveagent57 suggested just dropping them on the dashboard but into the courseware makes sense to me too

Reply:
Yeah, dropping them into the courseware is what the current DSC should already be doing today 😄

Reply:
Yeah, didn't realize we'd have the course run id here, so we can reliably determine the course home page to land on (left a similar comment below).